vmdb/config/vmdb.tmpl.yml

```diff
@@ -288,6 +288,7 @@ workers:
         :duration: 10.seconds
         :capacity: 50
         :amqp_port: 5672
+        :event_catcher_openstack_infra:
```

Comment:
Hmm, so yes it looks like this isn't the right way to nest it -- the build is complaining about the syntax here.